components/PageNavigation.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <b-navbar
+    class="navigation-container"
+    data-qa="main navigation"
+  >
+    <b-container class="p-0">
+      <b-navbar-nav>
+        <b-nav-item
+          v-for="nav in navigation"
+          :key="nav.name"
+          :to="localePath(nav.path)"
+          active-class="font-weight-bold"
+          variant="secondary"
+          data-qa="main navigation link"
+        >
+          {{ nav.name }}
+        </b-nav-item>
+      </b-navbar-nav>
+    </b-container>
+  </b-navbar>
+</template>
+
+<script>
+  export default {
+    computed: {
+      navigation() {
+        return this.$store.state.navigation.data;
+      },
+
+      i18n() {
+        return this.$store.state.i18n.locale;
+      }
+    },
+
+    watch: {
+      i18n() {
+        this.getNavigationData();
+      }
+    },
+
+    mounted() {
+      this.getNavigationData();
+    },
+
+    methods: {
+      async getNavigationData() {
+        return this.$store.dispatch('navigation/init');
+      }
+    }
+  };
+</script>
+
+<style lang="scss" scoped>
+  @import '../assets/scss/variables.scss';
+
+  .navigation-container {
+    background: $bodygrey;
+    padding-top: 0;
+    padding-bottom: 0;
+  }
```

Comment:
Can you take a look at this suggestion, Richard?

Reply:
Done

Reply:
Thanks!

Reply:
No, hang on, this is not right. Because you are using <b-navbar> there is a navbar class already. You can use it directly, no need to add it as a class. It is there now twice.